applications/ecallmgr/src/ecallmgr_originate.erl

```diff
@@ -883,7 +862,7 @@ should_update_uuid(OldUUID, Props) ->
         _ -> 'false'
     end.
 
--spec handle_fs_event(kzd_freeswitch:data(), created_uuid()) -> created_uuid().
+-spec handle_fs_event(kz_evt_freeswitch:data(), created_uuid()) -> created_uuid().
```

Comment:
This is still a `kz_proplist()` as far as I can tell. Why the spec change?

Reply:
enthusiasm :) `search/replace`. will fix

Reply:
Don't forget me!!!